### PR TITLE
Add index and create routes

### PR DESCRIPTION
**Pull request by anurag-majumdar**
1. Adding index and create routes.
2. Adding semicolons to places missing them inside controller/notes.js.

@skyshader 
Please review and let me know any changes to be made. Thanks. 😃

### CODE REVIEW

**Comment by VinayV9** [comment]
in we don't need semi coloumns nodejs

File: server/controllers/notes.js [hunk 1]
```diff
@@ -1,30 +1,59 @@
...
-        console.log(`delete: ${req.params}`)
-        res.send('ok')
+        console.log(`delete: ${req.params}`);
+        res.send('ok');
     },
 
     index: (req, res) => {
-       console.log(`index: ${req.body}`)
-       res.send('ok')
+        Note.find()
```
Comment (skyshader) [inline server/controllers/notes.js]: Let's only return non-deleted notes.

File: server/controllers/notes.js [hunk 2]
```diff
@@ -1,30 +1,59 @@
...
 
 const notesController = {
     create: (req, res) => {
-        // code here
-        console.log(`create: ${req.body}`)
-        res.send('ok')
+        const note = new Note({
+            title: req.body.title,
+            notes: req.body.notes
+        });
```
Comment (skyshader) [inline server/controllers/notes.js]: Let's follow one practice throughout. If we have semicolons everywhere, keep these. Of we don't remove these. :)

**Comment by anurag-majumdar** [comment]
Ok bro, making the respective changes and sharing them asap.

File: server/controllers/notes.js (hunk 1 above)
Comment (anurag-majumdar) [reply server/controllers/notes.js]: Will use the following `Note.find({ deleted: false })`

**Comment by anurag-majumdar** [comment]
@skyshader I have made the requested changes. Please review. Thanks. 😃

**Pull request merged**